scripts/lint-extras.sh

```diff
@@ -0,0 +1,93 @@
+#! /bin/bash
+# vim: set ts=4 sw=4 et :
+
```

Comment:
Add a description of the script in the header. Also include some example of how the script can be executed.